scripts/app.js

```diff
@@ -8,18 +8,23 @@ function Project (properties) {
   this.location = properties.location;
 }
```

Comment:
Let's try and refactor this so we aren't having to grab each property individually.